.circleci/config.yml

```
@@ -487,6 +503,51 @@ jobs:
           replace: $SNAPSHOT_VERSION
       - notify_github:
           data: "$(jq -Rcs '{ body: . }' scripts/snapshot-comment.md)"
+  build_check_android:
+    parameters:
+      working_directory:
+        type: string
+      android_path:
+        type: string
+    executor:
+      name: android/android-machine
+      tag: '2024.01.1'
+    steps:
+      - advanced-checkout/shallow-checkout
+      - build_and_pack_sdk:
+          working_directory: << parameters.working_directory >>
+      - run:
+          name: Build Android
+          command: |
+            cd << parameters.android_path >>
+            ./gradlew assembleRelease
```

Besides getting the Android path from the working directory parameter, I believe using CircleCI config's `working_directory` instead of `cd` in this case would be more readable. So in total I feel like it should look like this:
```suggestion
          command: ./gradlew assembleRelease
          working_directory: <<parameters.working_directory>>/android
```